### ISSUE TITLE
Publish to a single connection ID

### ISSUE THREAD

**Issue by aleclarson** [fastly/pushpin#47689]
Am I forced to subscribe the connection on `OPEN` to a `Grip-Channel` meant for only one connection before I can send a message via the `/publish` endpoint?

What if I could publish to a special `c:<id>` channel that maps to a single connection (without being forced to manually subscribe the connection to it first, of course)? :)

**Comment by jkarneges** [comment]
Correct. For publishing to specific connections, you could subscribe using the connection ID as the channel, but this is not done automatically.

**Comment by chrismendez** [comment]
Is there not a way to send a websocket text/message to a specific connection id? I can send a message to one particular connection id when responding on the REP spec, but not though any kind of control spec, is that correct?

**Comment by jkarneges** [comment]
That's correct. Control endpoints can send messages only to channels.

One thing you can do by connection ID is [refresh](https://pushpin.org/docs/advanced/#refresh). Then the backend can respond with any messages, including control messages.
